CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ target_compile_definitions(webview PUBLIC ${WEBVIEW_COMPILE_DEFS})
 target_compile_options(webview PRIVATE ${WEBVIEW_COMPILE_OPTS})
 target_link_libraries(webview ${WEBVIEW_LIBS})
 
-add_executable(webview_test WIN32 MACOSX_BUNDLE webview_test.cc)
+add_executable(webview_test MACOSX_BUNDLE webview_test.cc)
```

Comment:
I removed the WIN32 CMake declaration since webview_test.cc doesn't have a WinMain.